MapboxMobileEvents/MMENamespacedDependencies.h

```diff
@@ -425,6 +425,10 @@
 #define MMEEventTypeLocalDebug __NS_SYMBOL(MMEEventTypeLocalDebug)
 #endif
 
+#ifndef MMEEventTypeOfflineDownload
```

Comment:
What's this about? Is this related to CarPlay events at all?

Reply:
No, this is a separate event related to Maps SDK